src/components/Article.jsx

```diff
@@ -1,3 +1,65 @@
+import { useParams } from "react-router-dom"
+import {getArticleById, patchArticleById} from '../utils/API'
+import { useState,useEffect } from "react"
+import Like from '../icons/thumb-up.png'
+import dislike from '../icons/dislike.png'
+import '../styles/Article.css'
+import { Link } from "react-router-dom"
+
 export const Article = () =>{
-    
+    const {article_id} = useParams()
+    const [article,setArticle] = useState({})
+    const [like , setLike] = useState(0)
+    const [undo, setUndo] = useState(0)
+
+    useEffect(()=>{
+        getArticleById(article_id).then(article=>{
+            setArticle(article)})
+    },[article_id])
+
+    useEffect(()=>{
```

Comment:
Keep to one PR per ticket please - it makes it really hard to review otherwise and your feedback will be of a lower quality.